scripts/clippy.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# -e is not set here so that we can see the results of many separate
+# clippy runs, even if some of them have errors.
+set -uo pipefail
+
+# Required to correctly build SVM crates. We can use dummy values for the purposes of linting
+export BRIDGE_ADDRESS=""
+export EMITTER_ADDRESS=""
+export CHAIN_ID="1"
+export VERIFY_VAA_SHIM_PROGRAM_ID="abc"
+export POST_MESSAGE_SHIM_PROGRAM_ID="def"
+export TOKEN_BRIDGE_ADDRESS="ghi"
+
+# Go to repo root (in case script is called from another dir)
+# NOTE: Assumes this script is under `scripts/`
+cd "$(dirname "$0")/.." || exit
+
+# This should be the latest available Rust version for the system in order to match
+# our CI configuration.
+echo "Running Rust linting with active toolchain: $(rustup show active-toolchain)"
+
+# Ensure clippy component is installed for current active toolchain
+# This will use the toolchain defined in rust-toolchain* files via rustup
+# echo "Ensuring clippy component is installed for toolchain: $(rustup show active-toolchain)"
+# rustup component add clippy --toolchain "$(rustup show active-toolchain | awk '{print $1}')"
```

Comment:
Is this supposed to be commented out?

Reply:
Thanks, yeah it is. I didn't want to mess with the developer's set-up ultimately.